src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'chip version': version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
If `keyName === variableName` you can use shortened syntax.
Also, you don't need to write keys in `''` if it's valid variable name.
```suggestion
  const robot = {
    name,
    wheels,
    chipVersion,
    coords: {
      x: 0,
      y: 0,
    },
```